### DISCUSSION TITLE
Support for SDM230 seems to be broken

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#13481] [Q&A] [answered]
Hello
was running 9.3.1.0 for ESP32 and self compiled with SDM* enabled and the connection to the Eastron SDM230-Modbus device was working (SDMx20 Tx/Rx).
Tasmota 9.5 (and higher) seems to break this configuration and the Modbus connection. But configured as "SDM630 Tx/Rx" gets the Modbus connection up and running, but for 3 phases.

What have to be done to get the SDM230-Modbus device "up and running" with version 10.x ?

https://github.com/arendst/Tasmota/discussions/11504#discussioncomment-583884

Thanks, Norbert

**Comment by barbudor** [comment]
Can you confirm that v9.4.0 is working fine and that 9.5.0 is the 1st version from which it stopped working ?

It doesn't make sense.
I can't see any change in the driver since v9.3.1 until v9.5.0 which could have a behavior impact.
The only recent major change on energy is in 10.0 with the option to compute energy totals by phase.

**Reply by Noschvie** [reply]
For the SDM230 device the Modbus speed parameter have to be changed:

`#define SDM120_SPEED      9600    // default SDM120 Modbus address`

**Comment by Noschvie** [comment, marked as answer]
... it' working again, thanks!
